claws/openclaw/sageox-summary/SKILL.md

```
@@ -0,0 +1,552 @@
+   ```
+
+7. Verify `ox` is on the current subprocess PATH:
+
+   ```bash
+   command -v ox
+   ```
+
+   If it is **not** on PATH, print a copy-pasteable block tailored to the
+   user's detected paths and tell them to add it to `~/.openclaw/.env`:
+
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)